src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
```

Comment:
This code section is a Javadoc of the `login()` method and should be right above it, class fields on the other hand should be at the beginning of the class